core/templates/dev/head/pages/exploration_player/FeedbackPopupDirective.js

```diff
@@ -47,6 +49,10 @@ oppia.directive('feedbackPopup', [
           $scope.feedbackPopoverId = (
             'feedbackPopover' + Math.random().toString(36).slice(2));
 
+          if ( windowDimensionsService.getWidth() < 651 ) {
```

Comment:
Nit: remove spaces after '(' and before ')'

Can we make 651 a named constant defined in app.js? It appears in several places and we should make it such that it appears just once in the codebase. Alternatively we could write a custom function in windowDimensionsService to check narrowness for this purpose.